Gemfile.lock

```diff
@@ -483,7 +481,6 @@ DEPENDENCIES
   faker!
   graphiql-rails (= 1.4.11)
   graphql (= 1.9.21)
-  graphql-client
```

Comment:
wieso das hier verschwunden ist, ist mir etwas ein Rätsel

Reply:
should have been done with 0248d4e, i updated in master with 6e363d7 and rebased this branch to exclude that change in this PR